app/login/page.test.tsx

```diff
@@ -76,4 +79,34 @@ describe('Login', () => {
 
     mockUseAuthContext.isSignedIn = false;
   });
-});
+
+  test('successfully logs in and shows success notification', async () => {
+    const pretendUser = {
+      email: 'testemail@email.com',
+      password: 'test1234',
+    }
+
+    await mockLoginAccount(pretendUser);
+    expect(account.createEmailPasswordSession).toBeInstanceOf(Object);
+
+    render(<Alert variant={AlertVariants.Success} message="You've successfully logged in!" />)
```

Comment:
FYI, this is not testing, this is executing the component. We want to test that the component is on the screen with the variant and message we expect. In line is executing the component onto the screen.